### PR TITLE
Feature/kafka producer

### PR DESCRIPTION
**Pull request by crnopster**
Create a component for producing kafka messages to a configured topic - Done
Producer has to be synchronous - Done
Produced messages should have ability to be consumed in order from other side - Done

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -95,3 +96,24 @@ func TestConfigNewLoggerConfig(t *testing.T) {
...
 }
+
+func TestConfigNewKafkaConfig(t *testing.T) {
+	v, err := NewConfig("testConfigForViper", "./testData/")
+	if err != nil {
+		t.Fatalf("Cant start test, err: %v", err)
+	}
+
+	var partition int32
+	partition = 1
```
Comment (alpaka-alpachino) [inline config/config_test.go]: Is it necessary to declare variable in such way? Or can't it be const in that case?)

Comment (crnopster) [reply config/config_test.go]: fixed

**Review by alpaka-alpachino — DISMISSED** [review]

File: config/config_test.go [hunk 2]
```diff
@@ -6,6 +6,7 @@ import (
 	"testing"
```
Comment (oklymchuk) [inline config/config_test.go:6]: I would suggest to use github.com/stretchr/testify in tests instead of reflect.

File: kafka/producer.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package kafka
```
Comment (oklymchuk) [inline kafka/producer.go:2]: I would suggest to give short comment to package

File: config/config_test.go (hunk 2 above)
Comment (crnopster) [reply config/config_test.go:6]: Any reason to use another package?

File: kafka/producer.go (hunk 3 above)
Comment (crnopster) [reply kafka/producer.go:2]: fixed

**Review by alpaka-alpachino — DISMISSED** [review]

**Review by oklymchuk — DISMISSED** [review]

**Review by i3odja — DISMISSED** [review]

**Review by alpaka-alpachino — APPROVED** [review]

**Review by i3odja — APPROVED** [review]

**Review by oklymchuk — APPROVED** [review]

**Review by dmitriyminer — APPROVED** [review]

File: position/position.go [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+	"github.com/google/uuid"
+)
+
+type Position struct {
+	UserID    uuid.UUID // id of user that produced position notification event
+	Latitude  float32   // obviously latitude component of coordinates
+	Longitude float32   // obviously longitude component of coordinates
+	Timestamp time.Time // time when position collected
+	Arrival   time.Time // time when position accepted by system
```
Comment (radikh) [inline position/position.go:14]: Comments should not appear after fields, and should be in format: "FieldName ...".

File: kafka/producer.go [hunk 5]
```diff
@@ -0,0 +1,49 @@
+// Package kafka provides producer and consumer to work with kafka topics
```
Comment (radikh) [inline kafka/producer.go:1]: Any tests for the package?

**Review by radikh — CHANGES_REQUESTED** [review]
Lack of interfaces :laughing: 

Awaiting for changes that we discussed.

**Comment by crnopster** [comment]
kafka producer implemented in PR #23

**Pull request closed without merging**